cmake/modules/other-compileroptions.cmake

```diff
@@ -4,8 +4,24 @@ if (ENABLE_FAST_MATH)
     message(WARNING "Fast-Math might cause particle loss! Only use this if you know what you are doing!")
 endif()
 
-target_compile_options(autopas
-        PUBLIC
+if(ENABLE_CUDA)
```

Comment:
if all CUDA stuff is wrapped in generator expressions this if (and some others) are not needed.

Reply:
needed for cmake <3.11 see #184